mycli/main.py

```diff
@@ -1181,11 +1182,14 @@ def is_select(status):
 
 
 def thanks_picker(files=()):
+    contents = []
     for filename in files:
```

Comment:
Why not [fileinput](https://docs.python.org/3/library/fileinput.html)?

Reply:
I don't know, I only fixed this bug :smile: but you're right, fileinput would make sense! I'll see if I can fix this in another PR. Thanks for the suggestion!

Reply:
I don't know, I only fixed this bug :smile: but you're right, fileinput would make sense! I'll see if I can fix this in another PR. Thanks for the suggestion!